kolibri/core/assets/src/vue/error-box/index.vue

```diff
@@ -24,20 +25,48 @@
         error: state => state.core.error,
       },
     },
+    data: () => ({
+      isHidden: false,
+    }),
+    methods: {
+      hideErrorbox() {
+        this.isHidden = true;
+      },
+    },
   };
 
 </script>
 
 
 <style lang="stylus" scoped>
 
-  h1
+  .error-box-wrapper
+    padding: 10px
     margin-top: 50px
+    position: relative
+    background-color: #FFF6F6
+    border: 2px solid #e0b4b4
+    color: #9F3A38
```

Comment:
Note that we have existing colors defined for errors and error backgrounds. I think your colors look better, but we should try to achieve consistency and avoid hard-coding color values like this.

Might want to look where the existing colors in `coreTheme.styl` are used and see if they can be updated?